README.md

```diff
@@ -62,16 +62,16 @@ Summary of new features:
 
 + A core language based on "Quantitative Type Theory" which allows explicit
   annotation of erased types, and linear types.
-+ `let` bindings are now more expressive, and can be used to define pattern
++ `let` bindings are now more expressive, and can be used to define pattern-
   matching functions locally.
-+ Names which are in scope in a type are also always in scope in the body of
++ Names that are in scope in a type are also always in scope in the body of
```

Comment:
Not an error, according to [Merriam-Webster](https://www.merriam-webster.com/words-at-play/when-to-use-that-and-which) (sorry for switching dictionaries, I'm not sure how to look this up in OED).
The other way around (using 'that' for non-restrictive clauses) might have been more erroneous.